beep/protocol/maccor_to_biologic_mb.py

```diff
@@ -600,7 +691,7 @@ def maccor_ast_to_biologic_seqs(self, maccor_ast):
                     loop = unrolled_loop
                 else:
                     loop_seq = self._create_loop_seq(
-                        seq_num, loop_start_seq_num, num_loops
+                        seq_num, loop_start_seq_num, num_loops - 1
```

Comment:
Just to verify this is correct, can you confirm that if I have maccor loop, and the loop value is set to `5` that the steps inbetween the `do` and the `loop` step will run precisely `5` times?
```
<Do 1>
//looped
<Loop 1>
```
(I had guessed it would run 6 times similar to biologic but had not confirmed)